index.html

```diff
@@ -163,24 +163,47 @@ <h2>About me</h2>
                 </div>
             </section>
 
-             <div>
-                <h2 class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-                <br>
-                <div class="textareas">
-                <textarea name="name" id="" cols="41" rows="2">Full name</textarea>
-                <textarea name="e-mail" id="" cols="41" rows="2">E-mail address</textarea>
-                <textarea name="text" id="" cols="41" rows="10">Enter text here</textarea>
-                <div class="butt"><button class="btn" type="button"> Get in touch</button></div>
-                </div>
-            </div>
+            <section class="form-section">
+             <form action="https://formspree.io/f/mjvzwlqy" method="post">
+                <p class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+                <ul>
+                    
+                    <li >
+                        <label for="name">
+                            <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                        </label>
+                    </li>
+                    <li>
+                        <label for="mail">
+                            <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                        </label>
+
+                    </li>
+                    <li>
+                        <label for="msg">
+                            <textarea name="user_message" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                        </label>
+                    </li>
+                    <li class="button back-image">
+                        <button type="submit" class="btn">Get in touch</button>
+                    </li>
```

Comment:
- [ ] great hob so far, Kindly make the changes regarding the image at the form bottom and the social media icons.

| Figma Design | Your Design  |
| :---:   | :---: | 
|  ![Screenshot from 2022-09-28 09-58-02](https://user-images.githubusercontent.com/69077061/192710014-909d448a-9eb5-4e09-b31c-4d4aa73aef62.png) | ![Screenshot from 2022-09-28 09-59-11](https://user-images.githubusercontent.com/69077061/192710293-335b2854-3ca2-4601-a3ab-9f958d2cd461.png) |

Reply:
kindly explain more about the backgraound images because mine are showing in the folder

Reply:
Okay...in this template there are a lot of backgrounds on different section of the page. There is one also on the bottom of the form as you can see from the figma template. You simply add a background image to the form sectio and alight it to the bottom. Good luck 👍.

Reply:
https://user-images.githubusercontent.com/100031675/192722484-4a5a01ff-fe0a-4410-b5e0-8588e3c0ca5c.mp4

https://user-images.githubusercontent.com/100031675/192722621-0e006069-a447-4f63-bc6f-817801744b9a.mp4

Reply:
I see, this might happen for a lot of reason. Kindly try your site live with a tools like [htmlpreview](https://htmlpreview.github.io/) and once you see it there. Submit for the review. Everything should be fine 😊

Reply:
Please also make sure your images are pushed to the remote repository... This is the most common case.